Frontend/src/app/dashboard/dashboard.css

```
@@ -2389,15 +2447,28 @@
 
 /* Timeline body rows */
 .timeline-body {
+  --timeline-row-height: 64px;
   display: flex;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)